subjects/java/checkpoints/todo-list/README.md

```diff
@@ -0,0 +1,68 @@
+## TodoList
+
+### Instructions
+
+Write a class `Task` that has two attributes: `description` and `status`, with the corresponding getters and setters.
+
+Write a class called `TodoList` that represents a to-do list. The class should be able to hold a specified number of tasks, each with a description and a status. Include methods to add tasks, set a task's status, update task descriptions, and display all tasks in a neat format showing their descriptions and statuses.
+
+> The description length will be less or equal to 30 Character.When the list is full nothing more will be added.
```

Comment:
(minor thing) there should be a space after the `.` -> `... character. When ...`
Also, why `Character` is capitalized?

Reply:
character was capitalized by mistake.
fixed.